src/components/goal/Goal.tsx

```diff
@@ -43,45 +111,61 @@ const Goals = ({ goal }: GoalProps) => {
   }
 
   return (
-    <div>
-      <p className={$goal.description}>{goal.description}</p>
-
-      <div className={$goal.tab}>
-        <ul className={$goal.listNames}>
-          {subgoals?.map((subgoal) => (
-            <li key={subgoal.subgoal_id}>
-              <br />
-              <Subgoals subgoal={subgoal} />
-            </li>
-          ))}
-        </ul>
-        <br />
-        <form onSubmit={handleSubGoalSubmit} className={$goal.createInput}>
-          <input
-            type="text"
-            name="description"
-            placeholder="Subgoal description"
-            required
-          />
-          <input
-            type="text"
-            name="instructions"
-            placeholder="Instructions"
-            required
-          />
-          <input
-            type="number"
-            name="target_max_attempts"
-            placeholder="# of attempts"
-            min="1"
-            required
-          />
-
-          <button type="submit" className={$goal.createButton}>
-            Add SubGoal
-          </button>
-        </form>
+    <div className={$goal.goal}>
+      <div className={$goal.textContainer}>
+        <p className={$goal.description}>{goal?.description}</p>
+
+        {!expandSubgoals && (
+          <div style={{ marginTop: "8px", display: "flex", gap: "16px" }}>
+            <div className={$goal.subgoalCountBadge}>
+              <div className={$goal.subgoalCount}>
+                {subgoals?.length} active benchmark
+                {subgoals?.length !== 1 && "s"}
+              </div>
+            </div>
+            <div
+              style={{
+                display: "flex",
+                height: "40px",
+                padding: "10px 24px",
+                justifyContent: "center",
+                alignItems: "center",
+                gap: "8px",
+              }}
+            >
+              <Link
```

Comment:
I changed that to a button on my local branch, but this runs into another aesthetic issue (not sure how much time it's worth). Currently I have the entire goal acting as a link to the goal page, which is indicated on hover by changing the background color. It looks bad, especially with what it does to children. Is there a better way to deal with this? I can potentially make it so that just the goal text serves as a link, maybe underlines on hover?

Reply:
I think you're right in that the entire card should direct to that goal's page. I wouldn't worry about styling for hover or click right now if there's not a clear design for the distinction between a hovered vs a not-hovered card (for benchmarks or goals). Feel free to take screenshots of what you are seeing and post them in the main Compass channel and we can add it to January's work.

Reply:
Changed this to a button, all the styling is in the element and there's a comment that it's a stand-in for a tertiary button, so hopefully that's easy to replace when that component is built.